css/css-display/display-contents-accessibility.html

```diff
@@ -0,0 +1,321 @@
+    <search style="display: contents;" data-expectedrole="search" data-testname="search with display:contents has search role" class="ex">x</search>
+    <section aria-label="label" style="display: contents;" data-expectedrole="region" data-testname="section with aria-label and display:contents has region role" class="ex"></section>
+
+    <div role="banner" style="display: contents;" data-expectedrole="banner" data-testname="div with role banner and display:contents has banner role" class="ex">x</div>
+    <div role="navigation" style="display: contents;" data-expectedrole="navigation" data-testname="div with role navigation and display:contents has navigation role" class="ex">x</div>
+    <div role="complementary" style="display: contents;" data-expectedrole="complementary" data-testname="div with role complementary and display:contents has complementary role" class="ex">x</div>
+    <div role="main" style="display: contents;" data-expectedrole="main" data-testname="div with role main and display:contents has main role" class="ex">x</div>
+    <div role="contentinfo" style="display: contents;" data-expectedrole="contentinfo" data-testname="div with role contentinfo and display:contents has contentinfo role" class="ex">x</div>
+    <div role="region" aria-label="label" style="display: contents;" data-expectedrole="region" data-testname="div with role region, aria-label and display:contents has region role" class="ex">x</div>
+    <div role="form" aria-label="label" style="display: contents;" data-expectedrole="form" data-testname="div with role form, aria-label and display:contents has form role" class="ex">x</div>
+    <div role="search" style="display: contents;" data-expectedrole="search" data-testname="div with role search and display:contents has search role" class="ex">x</div>
+
+    <!-- Links -->
+    <a href="#" style="display: contents;" data-expectedrole="link" data-testname="link with display:contents has link role" class="ex">x</a>
+    <a href="#" role="link" style="display: contents;" data-expectedrole="link" data-testname="link with explicit link role and display:contents has link role" class="ex">x</a>
+    <div role="link" style="display: contents;" data-expectedrole="link" data-testname="div with link role and display:contents has link role" class="ex">x</div>
+    <div role="link" tabindex="0" style="display: contents;" data-expectedrole="link" data-testname="div with link role, tabindex=0 and display:contents has link role" class="ex">x</div>
+
+    <!-- Lists -->
+    <ul style="display: contents;" data-expectedrole="list" data-testname="ul with display:contents has list role (child li has display:contents)" class="ex">
+        <li style="display: contents;" data-expectedrole="listitem" data-testname="li as child of ul, both with display:contents, has listitem role" class="ex">x</li>
```

Comment:
I'm honestly surprised this one passes in WebKit given that there is no visible list indication (see [layout lists thread](https://twitter.com/cookiecrook/status/1337226933822603270?lang=en)), and I'm not sure why Chromium is treating this as `none`, but you could improve the test by adding `role=list` to the `<ul>` (bypassing the listitis functionality in WebKit, and doing something similar in Chromium?)

In case it seems like I'm trying to get you to only write tests that pass, I'm not... However, I think some of these failures are because there is implementation flexibility in these particular markup examples, and if you remove that flexibility, we'll get closer to understanding whether these can be counted as real WPT errors, or just underspecified implementation details.

Reply:
@cookiecrook Got it; I've added duplicate tests that each have explicit `role="list"` to hopefully account for the flexibility: https://github.com/web-platform-tests/wpt/pull/43740/commits/cfad82d83cfdb185796c0ab836f1fb7771b1bce5.

Reply:
I don't think it's reasonable to check in the first one given that we know it is/can/might trigger some heuristics that are undefined in the specs as an implementation detail. Your second ("duplicate") test can probably be checked in since it bypasses those.

Reply:
Got it, fixed as of: fb97fe9db208360ff58b96d4b70af2be24a1e6f4.